files/centos-8/sge-openssl.patch

```diff
@@ -0,0 +1,223 @@
+patch 845d213472a32d1f3e6a55b2e7f108ae6f2e871a
+Author: marco.schmidt@gmail.com
+Date:   Fri Nov  3 10:40:04 GMT 2017
+  * Fix 1618: openssl 1.1 ciphers
```

Comment:
Reminder to check with open source group for permission to distribute these files